Calibration/HcalAlCaRecoProducers/python/alcaIsoTracksFilter_cfi.py

```diff
@@ -9,11 +9,11 @@
                                    HBHERecHitLabel   = cms.InputTag("hbhereco"),
                                    TriggerEventLabel = cms.InputTag("hltTriggerSummaryAOD","","HLT"),
                                    TriggerResultLabel= cms.InputTag("TriggerResults","","HLT"),
-                                   Triggers          = cms.vstring("HLT_PFJet40","HLT_PFJet80","HLT_PFJet140","HLT_PFJet200","HLT_PFJet260","HLT_PFJet320","HLT_PFJet320","HLT_PFJet400"),
+                                   Triggers          = cms.vstring(),
```

Comment:
Is the purpose of this to really accept any trigger that populates the PD? Do you foresee to run the new stream on HLTPhysics as well during 2016?